A##-list-health.md

```diff
@@ -0,0 +1,59 @@
+# Add List endpoint to gRPC Health service
+
+* **Author(s):** Marcos Huck
+* **Approver:** TBD
+* **Status:** Draft  
+* **Implemented in:** TBD
+* **Last updated:** 2024-12-17  
+* **Discussion at:** TBD
+
+
+## Abstract
+
+This proposal introduces a new `List` RPC endpoint for the Health service, allowing clients to retrieve the statuses of all monitored services or a filtered list of specific services. This feature simplifies integration with status-reporting dashboards and enhances observability for microservices.
+
+## Background
+
+The existing Health service provides basic health check functionality but lacks a mechanism to retrieve a comprehensive list of all monitored services and their statuses. 
```

Comment:
One thing to keep in mind here is that the list of health services exposed by an application can change over the lifetime of the process.  New services can be added at any point, which means that the set of service names returned by the List method may change over the lifetime of the process.

Reply:
That seems reasonable, might be worth adding a note.